show/main.py

```diff
@@ -21,8 +21,15 @@
 from swsssdk import SonicV2Connector
 
 import mlnx
+from collections import OrderedDict
```

Comment:
This is a system import move above the line, `from pkg_resources import parse_version` (line12) to follow PEP8 standards:

> Imports should be grouped in the following order:
>
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
>
> You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
#done